config/dumps/config.env

```diff
@@ -1,5 +1,5 @@
 SPARQL_ENDPOINT=http://vfb-triplestore:8080/rdf4j-server/repositories/cap
 VFB_CONFIG=https://raw.githubusercontent.com/EBISPOT/cap-pipeline-config/main/config/prod/neo4j2owl-config.yaml
-DUMPS_SOLR="all preferred_roots deprecation_label image_names has_image"
+DUMPS_SOLR="Species Common_species"
 DUMPS_PDB="all preferred_roots deprecation_label has_image has_neuron_connectivity has_region_connectivity subclass_of_intersection ontology_metadata"
```

Comment:
It looks like this line should be updated as well, but not familiar with PDB...

Reply:
May as well make PDB match for now.  PDB = production neo4j database.  We are not using this component at present, but might do if there is a demand for ontology visualisations or browsable trees.

Reply:
in construct_all.sparql, should the prefix be updated?

It is currently:
PREFIX : <http://virtualflybrain.org#>

Should it be:
PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
PREFIX obo: <http://purl.obolibrary.org/obo/>

Reply:
Are we going to use construct_Species.sparql at the moment? Because I remember like using 10 pre-determined species first, am I right or I'm just mixing this something else we have spoken